test/tests/components/ws-manager/prebuild_test.go

```diff
@@ -190,7 +190,7 @@ func TestPrebuildWorkspaceTaskFail(t *testing.T) {
 
 const (
 	prebuildLogPath string = "/workspace/.gitpod"
-	prebuildLog     string = "'🍊 This task ran as a workspace prebuild'"
+	prebuildLog     string = "'⚡\bThis task ran as a workspace prebuild'"
```

Comment:
**question**: Does `\b` work? I couldn't test this with a prebuild in the preview environment. 🤷